projects/kserve/testing/test_e2e.py

```diff
@@ -620,6 +620,14 @@ def test_consolidated_model(consolidated_model, namespace=None):
 
     llm_load_test_args = config.ci_artifacts.get_config("tests.e2e.llm_load_test.args")
 
+    size_name = consolidated_model.get("testing")
+    if not size_name:
+        size_name = "small"
+    else:
+        size_name = size_name.get("size", "small")
+    llm_load_test_dataset_sample_args = config.ci_artifacts.get_config(f"tests.e2e.llm_load_test.dataset_size.{size_name}")
+    llm_load_test_args |= llm_load_test_dataset_sample_args
+
```

Comment:
Thanks @npalaska , looks nearly all good :)

can you change this:
```suggestion
    size_name = consolidated_model["testing"]["size"]

    llm_load_test_dataset_sample_args = config.ci_artifacts.get_config(f"tests.e2e.llm_load_test.dataset_size.{size_name}")
    llm_load_test_args |= llm_load_test_dataset_sample_args
```
and include this in `base.yaml` and `base-standalone-tgis.yaml`:
```
testing:
  size: small
```
(and you can remove the `testing.size.small` in the other files)

this removes the hard-coded default `small` property in the code

and just quick remark:
```
size_name = consolidated_model.get("testing")
size_name = size_name.get("size", "small")
```
this naming was not intuitive, should have been something like this, to make it more readable:
```
testing_props = consolidated_model.get("testing")
size_name = testing_props.get("size", "small")
```

Reply:
Fixed.